src/proto/project.proto

```diff
@@ -68,6 +76,7 @@ message UpdateProjectRequest {
   optional ProjectAccess access = 4; // The access of the project (such as public vs private)
   optional ProjectStatus status = 5; // The status of the project (such as active vs inactive)
   optional ProjectAsset file = 6;    // Project file for the project
+  optional ProjectType type = 7; // The type of project
```

Comment:
This one should stay as optional, since it's effectively a patch operation, and we'll want to differentiate the explicit "unspecified" case from the not-provided case.

Reply:
Addressed this with most recent change.